cli/types/index.d.ts

```diff
@@ -1099,6 +1111,7 @@ declare namespace Cypress {
      *    cy.wrap({foo: {bar: {baz: 1}}}).its('foo.bar.baz')
      */
     its<K extends keyof Subject>(propertyName: K, options?: Loggable): Chainable<Subject[K]>
+    its(propertyPath: string, options?: Loggable): Chainable

```

Comment:
The changelog said the typings of `its` is now more robust. This solution returns `any` for a deep path, and worse yet, loses all type safety. If you misspell a property, there is no longer an error. The subject will silently become an `any`. That's less robust.

Would the Cypress team be open to allowing an array of keys for a path? Doing that would allow type safe deeply nested paths.

```ts
cy.wrap({ a: { b: 2 }}).it's(['a', 'b']) // number
cy.wrap({ a: { b: 2 }}).it's('b') // error
cy.wrap({ a: { b: 2 }}).it's(['a', 'c']) // error
```